hack/keycloak-postgresql-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Should this have been committed?

Reply:
Yes, it should be added earlier. This might be helpful when applying a workaround for https://github.com/kubernetes/minikube/issues/3129

Reply:
Thanks @slaskawi